app/uk/gov/hmrc/mobiletaxcreditssummary/services/TaxCreditsSummaryService.scala

```diff
@@ -34,39 +37,74 @@ class LiveTaxCreditsSummaryService @Inject()(taxCreditsBrokerConnector: TaxCredi
 
   override def getTaxCreditsSummaryResponse(nino: Nino)(implicit hc: HeaderCarrier, ex: ExecutionContext): Future[TaxCreditsSummaryResponse] = {
     val tcNino = TaxCreditsNino(nino.value)
+    val now: LocalDate = LocalDateProvider.now
 
     def buildTaxCreditsSummary(paymentSummary: PaymentSummary): Future[TaxCreditsSummaryResponse] = {
-      def getChildrenAge16AndUnder: Future[Seq[Person]] = {
-        taxCreditsBrokerConnector.getChildren(tcNino).map(children =>
-          Child.getEligibleChildren(children))
+      def getChildrenAge16AndUnder: Future[Seq[Child]] =
+        taxCreditsBrokerConnector.getChildren(tcNino).map(children => Child.getEligibleChildren(children))
+
+      def createLocalDate(year: Int, month: Month, day: Int): LocalDate = LocalDate.of(year, month, day)
+
+      def isFtnaeDate(payment: FuturePayment): Boolean =
+        payment.paymentDate.isAfter(createLocalDate(now.getYear, Month.AUGUST, 31).atStartOfDay()) && payment.paymentDate.getYear == now.getYear
+
+      def hasAFtnaePayment(paymentSummary: PaymentSummary): Boolean =
+        paymentSummary.childTaxCredit.flatMap(payment => payment.paymentSeq.sortWith((a, b) => a.paymentDate.isAfter(b.paymentDate)).headOption) match {
+          case Some(payment) if isFtnaeDate(payment) => true
+          case _                                     => false
+        }
+
+      def getFtnaeLink(children: Seq[Child], paymentSummary: PaymentSummary): Option[String] = {
+        val hasFtnaePayment: Boolean = hasAFtnaePayment(paymentSummary)
+        Child.hasFTNAEChildren(children) match {
+          case _ if !hasFtnaePayment => None
+          case true if now.isBefore(createLocalDate(now.getYear, Month.SEPTEMBER, 1)) && hasFtnaePayment =>
+            Some("/tax-credits-service/home/children-and-childcare")
+          case true
+            if now.isAfter(createLocalDate(now.getYear, Month.AUGUST, 31)) &&
+              now.isBefore(createLocalDate(now.getYear, Month.SEPTEMBER, 8)) && hasFtnaePayment =>
+            Some("/tax-credits-service/children/add-child/who-do-you-want-to-add")
+          case _ => None
+        }
       }
 
-      val childrenFuture = getChildrenAge16AndUnder
-      val partnerDetailsFuture = taxCreditsBrokerConnector.getPartnerDetails(tcNino)
+      def getInformationMessage(children: Seq[Child]): Option[String] =
+        if (paymentSummary.specialCircumstances.isDefined){
+          val childChildren = Child.countFtnaeChildren(children) match {
+            case 0 | 1 =>  "child is"
+            case _ => "children are"
+          }
+          Some(s"We are currently working out your payments as your $childChildren changing their education or training. This should be done by 7 September ${LocalDateTime.now.getYear}. If your $childChildren staying in education or training, update their details on GOV.UK.")
+        }
+        else None
+
+      val childrenFuture        = getChildrenAge16AndUnder
```

Comment:
These futures are being executed in parallel, it's worth checking that any caching happening in downstream services won't be impacted by this like it was in TAI. Maybe we don't want to execute in parallel, but sequentially

Reply:
This is how it was before. I have just rearranged slightly...

Reply:
Ah ok - not to worry then, just thought I'd mention it in case, didn't want to have #DuplicateCallGate like we did in tai 🙈